modules/aggregation-layers/src/heatmap-layer/heatmap-layer.js

```diff
@@ -276,18 +282,18 @@ export default class HeatmapLayer extends AggregationLayer {
     }
   }
 
+  _getWeightsTransformShaders() {
```

Comment:
Can you use `getShaders` for both transforms? Similar to:

https://github.com/visgl/deck.gl/blob/cc225a850e343b5aadd1eb36b67b1e8fac17b66d/modules/layers/src/solid-polygon-layer/solid-polygon-layer.js#L64